Gems/ROS2/Code/Source/Lidar/LidarTemplateUtils.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "LidarTemplateUtils.h"
+#include <AzCore/Math/MathUtils.h>
+#include <AzCore/Math/Matrix4x4.h>
+#include <AzCore/Math/Quaternion.h>
+#include <AzCore/Math/Vector3.h>
+#include <AzCore/Utils/Utils.h>
+#include <AzCore/std/containers/unordered_map.h>
+
+namespace ROS2
+{
+    LidarTemplate LidarTemplateUtils::GetTemplate(LidarTemplate::LidarModel model)
+    {
+        static std::unordered_map<LidarTemplate::LidarModel, LidarTemplate> templates;
```

Comment:
With the allocator availability work done by @burelc-amzn, AZStd::unordered_map can now be used at static scope.
```suggestion
        static AZStd::unordered_map<LidarTemplate::LidarModel, LidarTemplate> templates;
```